solution/deps.yaml

```diff
@@ -80,8 +80,14 @@ s3utils:
   sourceRegistry: ghcr.io/scality
   dashboard: s3utils/s3utils-dashboards
   image: s3utils
-  tag: 1.14.5
+  tag: 1.14.6
   envsubst: S3UTILS_TAG
+scuba:
+  sourceRegistry: ghcr.io/scality
+  dashboard: scuba/scuba-dashboards
+  image: scuba
+  tag: 1.0.0-preview.2
```

Comment:
Note for reviewers: do we want a 1.0.0 tag? Not sure if it makes sense for a Scuba point of view, as it's still under dev...

Reply:
I would rather not have a preview release in production...
if we don't want to make a 1.0 release, I guess a `0.x.y` release or something like this may be acceptable, should be discussed with Scuba team...

Reply:
Asked Scuba team, I would rather prefer having a 1.0.0 as the project had this package version

Reply:
We are aligned, I created a `1.0.0` release (now used in the PR)